src/Circle.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace Shape;
+
+
+class Circle  implements ShapeInterface
```

Comment:
Hm. It looks like a lot of code here is duplicated from Square class. Duplicating code which serves for the same purpose in different places is never a good idea.
There is a way how to solve this problem - special kind of classes which can behave both as class and as interface by having some functionality implementation inside and just a methods templates like interfaces. I'll leave that up to you to find what is that, but in case you'll struggle a lot here - ping me, I'll help.